<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxConnexionServeur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>531</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion au serveur </string>
   </property>
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditEtatConnexion">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>191</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCOnnexion">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>298</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Adresse IP du serveur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Num√©ro de port du serveur</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditAdresseServeur">
         <property name="text">
          <string>172.18.58.83</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditNumeroPort">
         <property name="text">
          <string>1025</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxInformationCLient">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations Client</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>501</width>
      <height>118</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonNomDeLordinateur">
         <property name="text">
          <string>Nom de l'ordinateur distant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonNomUtilisateur">
         <property name="text">
          <string>Nom de l'utilisateur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonArchitectureProcesseur">
         <property name="text">
          <string>Architecture processeur</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOSOrdinateur">
         <property name="text">
          <string>OS de l'ordinateur</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEditNomDeLordinateur"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditNomUtilisateur"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditArchitectureProcesseur"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditOsOrdinateur"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
